templates/about-us/about-us.css

```diff
@@ -1,4 +1,8 @@
 .about-us {
+  h1 {
+    text-align: center;
```

Comment:
Do we still need this if the global heading style is defined at the template level?

Reply:
I have removed this.  👍